rules/docker-fpm-gobgp.mk

```diff
@@ -9,3 +9,4 @@ SONIC_DOCKER_IMAGES += $(DOCKER_FPM_GOBGP)
 $(DOCKER_FPM_GOBGP)_CONTAINER_NAME = bgp
 $(DOCKER_FPM_GOBGP)_RUN_OPT += --privileged -t
 $(DOCKER_FPM_GOBGP)_RUN_OPT += -v /etc/sonic:/etc/sonic:ro
+$(DOCKER_FPM_GOBPG)_BASE_IMAGE_FILES += $(SUPERVISOR_PROC_EXIT_LISTENER_SCRIPT)
```

Comment:
Check builds are failing because `SUPERVISOR_PROC_EXIT_LISTENER_SCRIPT` should be added to `$(DOCKER_FPM_GOBPG)_FILES`, not `$(DOCKER_FPM_GOBPG)_BASE_IMAGE_FILES`

Reply:
Fixed.